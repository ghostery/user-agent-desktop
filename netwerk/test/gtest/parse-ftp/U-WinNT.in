ftp> open ftp.microsoft.com
220 Microsoft FTP Service
Name (ftp.microsoft.com:cyp): 
331 Anonymous access allowed, send identity (e-mail name) as password.
230-This is FTP.Microsoft.Com
230 Anonymous user logged in.
Remote system type is Windows_NT.
ftp> quote dirstyle
200 MSDOS-like directory output is on
ftp> quote dirstyle
200 MSDOS-like directory output is off
ftp> ls
200 PORT command successful.
150 Opening ASCII mode data connection for /bin/ls.
dr-xr-xr-x   1 owner    group               0 Feb 25  2000 channel
dr-xr-xr-x   1 owner    group               0 Feb 25  2000 Education
dr-xr-xr-x   1 owner    group               0 Feb 25  2000 enterprise
dr-xr-xr-x   1 owner    group               0 Jun 20  2001 ISN
dr-xr-xr-x   1 owner    group               0 May 31  2001 Museum
dr-xr-xr-x   1 owner    group               0 Feb 14  2001 TechNet
dr-xr-xr-x   1 owner    group               0 Oct 24  2001 whql
dr-xr-xr-x   1 owner    group               0 Feb  5  2001 20Year
dr-xr-xr-x   1 owner    group               0 Sep 26  2000 AccessFoxPro
dr-xr-xr-x   1 owner    group               0 Dec 21  2000 AlbGrp
dr-xr-xr-x   1 owner    group               0 May 31  2001 Alexandra
dr-xr-xr-x   1 owner    group               0 Jan 19  2001 anna
dr-xr-xr-x   1 owner    group               0 Apr  6  2000 anz
dr-xr-xr-x   1 owner    group               0 May 10  2000 Chase Bobko2
dr-xr-xr-x   1 owner    group               0 Mar 29  2000 cnn
dr-xr-xr-x   1 owner    group               0 Nov 21  2000 Darin
dr-xr-xr-x   1 owner    group               0 Mar  9  2000 digitalchicago
dr-xr-xr-x   1 owner    group               0 Sep  6  2000 Dublin
dr-xr-xr-x   1 owner    group               0 Jan 30  2001 eleanorf
dr-xr-xr-x   1 owner    group               0 Apr 26  2001 girvin
dr-xr-xr-x   1 owner    group               0 Apr 26  2000 Hires
dr-xr-xr-x   1 owner    group               0 Aug 15  2000 HR
-r-xr-xr-x   1 owner    group         4368384 Oct 24  1999 IMG00022.PCD
dr-xr-xr-x   1 owner    group               0 May 18  2001 jacubowsky
dr-xr-xr-x   1 owner    group               0 Oct 12  2000 JFalvey
dr-xr-xr-x   1 owner    group               0 Mar 28  2001 johnci
dr-xr-xr-x   1 owner    group               0 Jul 14  2000 Karin
dr-xr-xr-x   1 owner    group               0 Sep  7  2000 Kjung
dr-xr-xr-x   1 owner    group               0 Sep 28  2000 LarryE
dr-xr-xr-x   1 owner    group               0 Aug 17  2000 Larson
dr-xr-xr-x   1 owner    group               0 Sep 12  2000 marion
dr-xr-xr-x   1 owner    group               0 Aug  9  2000 ms25
dr-xr-xr-x   1 owner    group               0 Nov 16  2000 MS25Brochure
dr-xr-xr-x   1 owner    group               0 Mar 29  2000 MShistory
dr-xr-xr-x   1 owner    group               0 Sep  5  2000 Neils
dr-xr-xr-x   1 owner    group               0 Aug  2  2000 NLM
dr-xr-xr-x   1 owner    group               0 Sep  6  2000 PageOne
dr-xr-xr-x   1 owner    group               0 Jun 27  2000 pccomputing
dr-xr-xr-x   1 owner    group               0 May  9  2001 pictures
dr-xr-xr-x   1 owner    group               0 Jul 21  2000 pranks
dr-xr-xr-x   1 owner    group               0 Aug 22  2000 Sean
dr-xr-xr-x   1 owner    group               0 Aug 10  2000 SLeong
dr-xr-xr-x   1 owner    group               0 Sep  7  2000 svr
dr-xr-xr-x   1 owner    group               0 Jul 21  2000 Transcontinental
dr-xr-xr-x   1 owner    group               0 Oct 23  2000 veronist
dr-xr-xr-x   1 owner    group               0 Jun 15  2000 zoe
-r-xr-xr-x   1 owner    group         2094926 Jul 14  2000 canprankdesk.tif
-r-xr-xr-x   1 owner    group           95077 Jul 21  2000 Jon Kauffman Enjoys the Good Life.jpg
-r-xr-xr-x   1 owner    group           52275 Jul 21  2000 Name Plate.jpg
-r-xr-xr-x   1 owner    group         2250540 Jul 14  2000 Valentineoffprank-HiRes.jpg
226 Transfer complete.
ftp> close
221  Thank-You For Using Microsoft Products!
ftp> 
