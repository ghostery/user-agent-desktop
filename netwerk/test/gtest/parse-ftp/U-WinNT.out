<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

02-25-2000  00:00:00       <DIR>          channel
02-25-2000  00:00:00       <DIR>          Education
02-25-2000  00:00:00       <DIR>          enterprise
06-20-2001  00:00:00       <DIR>          ISN
05-31-2001  00:00:00       <DIR>          Museum
02-14-2001  00:00:00       <DIR>          TechNet
10-24-2001  00:00:00       <DIR>          whql
02-05-2001  00:00:00       <DIR>          20Year
09-26-2000  00:00:00       <DIR>          AccessFoxPro
12-21-2000  00:00:00       <DIR>          AlbGrp
05-31-2001  00:00:00       <DIR>          Alexandra
01-19-2001  00:00:00       <DIR>          anna
04-06-2000  00:00:00       <DIR>          anz
05-10-2000  00:00:00       <DIR>          Chase Bobko2
03-29-2000  00:00:00       <DIR>          cnn
11-21-2000  00:00:00       <DIR>          Darin
03-09-2000  00:00:00       <DIR>          digitalchicago
09-06-2000  00:00:00       <DIR>          Dublin
01-30-2001  00:00:00       <DIR>          eleanorf
04-26-2001  00:00:00       <DIR>          girvin
04-26-2000  00:00:00       <DIR>          Hires
08-15-2000  00:00:00       <DIR>          HR
10-24-1999  00:00:00              4368384 IMG00022.PCD
05-18-2001  00:00:00       <DIR>          jacubowsky
10-12-2000  00:00:00       <DIR>          JFalvey
03-28-2001  00:00:00       <DIR>          johnci
07-14-2000  00:00:00       <DIR>          Karin
09-07-2000  00:00:00       <DIR>          Kjung
09-28-2000  00:00:00       <DIR>          LarryE
08-17-2000  00:00:00       <DIR>          Larson
09-12-2000  00:00:00       <DIR>          marion
08-09-2000  00:00:00       <DIR>          ms25
11-16-2000  00:00:00       <DIR>          MS25Brochure
03-29-2000  00:00:00       <DIR>          MShistory
09-05-2000  00:00:00       <DIR>          Neils
08-02-2000  00:00:00       <DIR>          NLM
09-06-2000  00:00:00       <DIR>          PageOne
06-27-2000  00:00:00       <DIR>          pccomputing
05-09-2001  00:00:00       <DIR>          pictures
07-21-2000  00:00:00       <DIR>          pranks
08-22-2000  00:00:00       <DIR>          Sean
08-10-2000  00:00:00       <DIR>          SLeong
09-07-2000  00:00:00       <DIR>          svr
07-21-2000  00:00:00       <DIR>          Transcontinental
10-23-2000  00:00:00       <DIR>          veronist
06-15-2000  00:00:00       <DIR>          zoe
07-14-2000  00:00:00              2094926 canprankdesk.tif
07-21-2000  00:00:00                95077 Jon Kauffman Enjoys the Good Life.jpg
07-21-2000  00:00:00                52275 Name Plate.jpg
07-14-2000  00:00:00              2250540 Valentineoffprank-HiRes.jpg
